jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/OdePropertiesTest.java

```diff
@@ -72,24 +72,24 @@ public void testInit() {
             }
       }
       
-      @Test
-      public void initShouldCatchUnknownHostException(@Capturing InetAddress capturingInetAddress) throws Exception {
-            // from jmockit dev history (https://jmockit.github.io/changes.html)
-            // as of versiofn 1.48, partial mocking of classes through Expectations has been
-            // dropped. MockUp is recommended alternative
-            new MockUp<InetAddress>() {
-                  @Mock
-                  public InetAddress getLocalHost() throws UnknownHostException {
-                        throw new UnknownHostException("testException123");
-                  }
-            };
+      // @Test
```

Comment:
Should we just remove instead of comment?

Reply:
This has been removed